config/symbols.pspeu.stlib.txt

```diff
@@ -1,3 +1,6 @@
+D_80097928 = 0x091CE5B0;
+D_80073510 = 0x091E17B8;
+D_8003BE5E = 0x092347A2;
```

Comment:
These are all outside the overlay, should be in `symbols.pspeu.txt` I think?

I think `0x091E17B8` would be `g_Entities[1].ext.something` but unclear what